tmt/utils.py

```diff
@@ -4763,6 +4763,7 @@ def git_clone(
         env: Optional[EnvironmentType] = None,
         shallow: bool = False,
         can_change: bool = True,
+        retry_times: int = 9
```

Comment:
1. the default value deserves to be a global "constant", e.g. see https://github.com/teemtee/tmt/blob/main/tmt/log.py#L445
2. `retries` sounds better to me, `retry_times` looks like it's related to timing.

Reply:
Nice comment, I didn't know how to name it actually. Use `retries` instead :-)